src/screens/InputUsername/index.js

```diff
@@ -343,97 +344,97 @@ export default ChooseUsername;
+    marginBottom: dimen.normalizeDimen(20)
   },
   input: {
-    borderWidth: 1,
-    borderRadius: 8,
+    borderWidth: dimen.normalizeDimen(1),
+    borderRadius: dimen.normalizeDimen(8),
     borderColor: '#BDBDBD',
-    paddingHorizontal: 23,
-    paddingVertical: 13,
+    paddingHorizontal: dimen.normalizeDimen(23),
+    paddingVertical: dimen.normalizeDimen(13),
     width: '100%'
   },
   container: {
     flex: 1,
     backgroundColor: colors.white
   },
-  btnNext: {marginTop: 16},
+  btnNext: {marginTop: dimen.normalizeDimen(16)},
   gap: {flex: 1},
   icon: {
-    width: 14,
-    height: 14,
+    width: dimen.normalizeDimen(14),
+    height: dimen.normalizeDimen(14),
     position: 'absolute',
-    bottom: -5,
-    left: 19
+    bottom: dimen.normalizeDimen(-5),
+    left: dimen.normalizeDimen(19)
   },
   image: {
-    height: 52,
-    width: 52,
-    borderRadius: 26
+    height: dimen.normalizeDimen(52),
+    width: dimen.normalizeDimen(52),
+    borderRadius: dimen.normalizeDimen(26)
   },
   title: {
     fontFamily: 'Inter-Bold',
     fontStyle: 'normal',
     fontWeight: '600',
-    fontSize: 36,
-    lineHeight: 43.57,
+    fontSize: normalizeFontSize(36),
+    lineHeight: normalizeFontSize(43.57),
     color: '#11243D',
-    marginTop: 24
+    marginTop: dimen.normalizeDimen(24)
   },
   desc: {
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: '400',
-    fontSize: 14,
-    lineHeight: 24,
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(24),
     color: colors.gray,
     opacity: 0.84,
-    marginTop: 8,
-    marginBottom: 12
+    marginTop: dimen.normalizeDimen(8),
+    marginBottom: dimen.normalizeDimen(12)
   },
   btnImage: {
-    width: 23,
-    height: 23,
+    width: dimen.normalizeDimen(23),
+    height: dimen.normalizeDimen(23),
     justifyContent: 'center',
     alignItems: 'center'
   },
   constainerInfo: {
     backgroundColor: 'rgba(47, 128, 237, 0.2)',
     flexDirection: 'row',
-    borderRadius: 8,
+    borderRadius: dimen.normalizeDimen(8),
     width: '100%',
-    paddingHorizontal: 7,
-    paddingVertical: 8
+    paddingHorizontal: dimen.normalizeDimen(7),
+    paddingVertical: dimen.normalizeDimen(8)
   },
   infoText: {
     fontFamily: 'Inter-Regular',
     fontStyle: 'normal',
     fontWeight: '400',
-    fontSize: 14,
+    fontSize: normalizeFontSize(14),
     color: COLORS.blue,
     // marginLeft: 12,
-    lineHeight: 24,
-    paddingHorizontal: 4
+    lineHeight: normalizeFontSize(24),
+    paddingHorizontal: dimen.normalizeDimen(4)
     // width: width - 95,
     // backgroundColor: 'red'
   },
   containerAddIcon: {
-    marginRight: 13
+    marginRight: dimen.normalizeDimen(13)
   },
   content: {
-    paddingTop: 20,
-    paddingHorizontal: 20
+    paddingTop: dimen.normalizeDimen(20),
+    paddingHorizontal: dimen.normalizeDimen(20)
   },
   keyboardavoidingview: {
     flex: 1,
     justifyContent: 'flex-end'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. By using `dimen.normalizeDimen` and `normalizeFontSize`, the dimensions and font sizes will be adjusted based on the screen size, which can help to ensure that the layout and text look consistent across different devices.



<!-- This is an auto-generated comment by OSS CodeRabbit -->